python/looker/rtl/user_session.py

```diff
@@ -10,11 +11,16 @@
 class UserSession():
     def __init__(self,
                  settings: st.ApiSettings,
-                 transport: Optional[tp.Transport] = None):
+                 transport: Optional[tp.Transport] = None,
```

Comment:
@josephaxisa I wonder if maybe your mypy <=> ide integration isn't working? It also breaks whenever `ml` is regenerated with the current syntax error. If your ide isn't showing type errors, try just running `mypy looker/rtl/user_session.py`

I didn't get to fix all the type stuff in this module, I left some comments in the code. Feel free to take a stab at them